__tests__/pages/provinces/index.test.tsx

```diff
@@ -8,6 +8,17 @@ import ProvincesPage, { getStaticProps } from "~/pages/provinces";
 
 import { render, screen, within } from "@testing-library/react";
 
+jest.mock("next/router", () => ({
+  useRouter() {
+    return {
+      route: "/provinces",
+      pathname: "/provinces",
+      query: {},
+      asPath: "/provinces",
+    };
+  },
+}));
```

Comment:
Is replacing this with `next-router-mock` breaking your test?
If not, please replace it, so we don't need to maintain the mocking code too much. 😁